src/components/UserInfo/UserInfo.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { User } from '../../types/types';
+import './UserInfo.scss';
+
+interface Props {
+  user: User | null;
```

Comment:
Цей компонент рендериться, коли вже точно є user
```suggestion
  user: User;
```